packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -698,11 +603,15 @@ test.serial('writeGranules() writes all valid files if any non-valid file fails'
   t.false(await filePgModel.exists(knex, { key: invalidFiles[0].key }));
   t.false(await filePgModel.exists(knex, { key: invalidFiles[1].key }));
 
-  const fileRecords = await filePgModel.search(knex, {});
+  const granuleCumulusId = await granulePgModel.getRecordCumulusId(
```

Comment:
This method on the granule model should also force you to pass in the minimal required fields to identify a granule record uniquely in PG, but I'm not gonna put that on you in this PR

Reply:
probably getRecordsCumulusId too...